### DISCUSSION TITLE
change behiver of a device

### DISCUSSION THREAD

**Discussion by zeev-mindali** [zwave-js/zwave-js#6689] [Q&A] [answered]
Dear all,

i have a device, for some reason report himself as a light instead of cover, as a multiLevel switch.
can i change the the up button to send a specific value and the down button to another value?
it will be much easier to handle on zwave rather then build entire helpers and entites in home assistant.

**Comment by AlCalzone** [comment, marked as answer]
I'm afraid that's not possible.
